test/ex_wrapplog_test.exs

```diff
@@ -6,46 +6,55 @@ defmodule ExWrapplogTest do
   test "log method" do
     Application.put_env(:ex_wrapplog, :level, :debug)
     assert capture_io(fn  -> ExWrapplog.log(:error, "This is an error") end)
-      == "ERROR {\"msg\":\"This is an error\",\"level\":\"error\"}\n"
+      == "{\"timestamp\":\"2017-03-28T13:46:06.989172Z\",\"service\":\"\",\"msg\":\"This is an error\",\"level\":\"error\"}\n"
```

Comment:
Yes the order should not matter you are right. But here I want to test the format of JSON and the fields which are present in JSON. If I add a flexible tests here then I wont be able to test that. SO its a tradeoff.